common/main/java/com/couchbase/lite/Collection.java

```diff
@@ -0,0 +1,189 @@
+package com.couchbase.lite;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+
+import java.util.Date;
+import java.util.List;
+import java.util.concurrent.Executor;
+
+import com.couchbase.lite.internal.utils.Preconditions;
+
+
+public class Collection implements Indexable, DatabaseChangeObservable {
```

Comment:
final?

Reply:
following PMD rules of naming and declaring variables

Reply:
The class.  Could the class be final?